helpers/bootrr-generic-tests

```diff
@@ -2,3 +2,12 @@
 
 assert_file_is_empty deferred-probe-empty /sys/kernel/debug/devices_deferred
 
+test_cpu_bringup() {
```

Comment:
To be more coherent with current layout, I'd add this test in a separate assert_cpu_bringup file following the same schema as others, and here just call 

assert_cpu_bringup 

If we start to add generic functions for tests here, this file can end up unreadable.

Reply:
Apart from this, the test LGTM. But did you try with a kernel that has CONFIG_SMP=n? It is a common configuration on different build. I'm wondering if this just passes or give us a false positive.